my-app/src/App.css

```diff
@@ -13,7 +13,7 @@
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
+  //font-size: calc(10px + 2vmin);
```

Comment:
This is not an actual CSS comment, right? `/* font-size: calc(....) */` would be a CSS comment.

> CSS uses the same "block comment" syntax as the C-like languages - you start a comment with /* , and end it with */ . However, CSS is missing the "line comment" syntax that those languages have, where everything from // to the end of the line is commented out.

Reply:
I don't know why this  was added, I have removed it now

Reply:
PS: Your work is awesome, there's lots of activity, hard to even keep up. You rock.

Reply:
Thanks Olle :-)